src/NewCommand.php

```diff
@@ -81,6 +107,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }, $commands);
         }
 
+        if ($input->getOption('with')) {
```

Comment:
Here you can do `if ($with = $input->getOption('with')) {` and replace all `$input->getOption('with')` bellow with `$with`.